Decisions.Equifax/PrequalificationOfOne/Response/PreQualificationOfOneResponse.cs

```diff
@@ -0,0 +1,750 @@
+﻿using System.Runtime.Serialization;
+using DecisionsFramework.Design.ConfigurationStorage.Attributes;
+using Newtonsoft.Json;
+
+namespace Decisions.Equifax.PreQualificationOfOne.Response
+{
+  
+        [DataContract]
+        [Writable]
+        public class PreQualificationOfOneResponse
```

Comment:
Since almost all of this is duplicated. We should create a common class and just override the pieces that we need to in a sub class so that we're not duplicating all this code. For example, the common class could have things like the `Status` property and it could be used in both the `ConsumerCreditReportResponse` and this class.